### DISCUSSION TITLE
EPIC Frequently Asked Questions

### DISCUSSION THREAD

**Discussion by support-epic** [ufs-community/ufs-srweather-app#563] [General]
Check out our technical FAQs page to see if your question has already been asked: https://epic.noaa.gov/technical-faqs/

**Comment by DavidHuber-NOAA** [comment]
For which grid and for how many forecast hours is the cloud cost estimate applicable?

**Reply by ShawnCebulaNOAA** [reply]
Hello David!
I just checked with our team, including our Chief Engineer on this for you - the cloud cost estimate is effectively applicable to anything you'd run on AWS as their prices are based on the compute resources used, i.e. the computing cluster size and time, with some added cost based on the amount of data uploaded/downloaded.

**Comment by DavidHuber-NOAA** [comment]
Hmm, that doesn't make sense to me.  I would assume that the compute resources needed for a 3km CONUS grid would be ~ 16 times greater than those needed for a 13km CONUS grid.  Thus my question about which grid was used.  If this is for a 13km grid, then I can probably assume that if I needed to run the 3km grid, the compute costs would be 16 times larger since it would either require more nodes or more compute hours to compensate.

**Reply by ShawnCebulaNOAA** [reply]
That's generally the case with a bit of leeway either way for data transfer and staging costs, sorry if I wasn't clear, it should be grid-agnostic since it's based entirely on time and compute power of the cluster type you choose.

**Comment by kevenmblackman** [comment]
Hello David: 

That is correct, the 3km would be considerably more expensive than the 13km.  We had a SRW run to support quite a few community events this year doing the following. This is what the costing is based on.

Run 3 experiments:
6 hour 25 km CONUS forecast for 6/15/2019
6 hour high resolution forecast over Indianapolis with GFSv16 physics
6 hour high resolution forecast over Indianapolis with RRFS physics 

You bring up a good point that we need to absolutely call this out to the exact use case detailing forecast hours and grids for this cost outline.

**Reply by DavidHuber-NOAA** [reply]
Thank you for clarifying and for providing the costing cases, Keven.  That's very informative.
